### ISSUE TITLE
Error npm run build for dev install

### ISSUE THREAD

**Issue by poka-IT** [Chocobozzz/PeerTube#1115]
I'm trying to start another peertube instance in dev mode (in a fresh debian 9 VM), but first, I don't find the dev install documentation anymore, not in github, but I found it few weeks ago ...

And don't understand exactly where the problem from, but I can launch a 

> npm run watch:client

Server side occure problems. I past logs at the end of ticket.

* **PeerTube version or URL:** beta 13

* **NodeJS version:** v8.11.3

* **What is the expected behaviour?**
Load peertube instance in dev mode
* **What do you see instead?**
Errors at the end of 

> npm run build

 or 

> npm run dev

* **Server log:**

> npm run build

```
[1] 17:04:54 - Starting compilation in watch mode...
[0] > peertube@1.0.0-beta.13 nodemon /home/peertube/PeerTube
[0] > nodemon "--delay" "2" "--watch" "./dist" "dist/server"
[0] 
[0] [nodemon] 1.18.4
[0] [nodemon] to restart at any time, enter `rs`
[0] [nodemon] watching: /home/peertube/PeerTube/dist/**/*
[0] [nodemon] starting `node dist/server`
[0] [nodemon] Internal watch failed: watch /home/peertube/PeerTube/dist ENOSPC
[0] npm ERR! code ELIFECYCLE
[0] npm ERR! errno 1
[0] npm ERR! peertube@1.0.0-beta.13 nodemon: `nodemon "--delay" "2" "--watch" "./dist" "dist/server"`
[0] npm ERR! Exit status 1
[0] npm ERR! 
[0] npm ERR! Failed at the peertube@1.0.0-beta.13 nodemon script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
[0] 
[0] npm ERR! A complete log of this run can be found in:
[0] npm ERR!     /home/peertube/.npm/_logs/2018-09-24T17_05_08_945Z-debug.log
[0] npm run tsc -- --sourceMap && npm run nodemon -- --delay 2 --watch ./dist dist/server exited with code 1
--> Sending SIGTERM to other processes..
[1] npm run tsc -- --sourceMap --preserveWatchOutput -w exited with code SIGTERM
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! peertube@1.0.0-beta.13 concurrently: `concurrently "-k" "npm run tsc -- --sourceMap && npm run nodemon -- --delay 2 --watch ./dist dist/server" "npm run tsc -- --sourceMap --preserveWatchOutput -w"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the peertube@1.0.0-beta.13 concurrently script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/peertube/.npm/_logs/2018-09-24T17_05_09_083Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! peertube@1.0.0-beta.13 watch:server: `scripty`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the peertube@1.0.0-beta.13 watch:server script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/peertube/.npm/_logs/2018-09-24T17_05_09_129Z-debug.log
```

**Comment by rigelk** [comment]
Hi @poka-IT - glad your are trying to develop on PeerTube!

The documentation to run it in development is https://github.com/Chocobozzz/PeerTube/blob/develop/.github/CONTRIBUTING.md

As for your current problem, you might be looking for [a similar answer to this](https://stackoverflow.com/a/32600959).

**Comment by poka-IT** [comment]
So I installed peertube in dev mode on my local laptop, it's ok with no problem for server and client.

Still don't understand why I still have the problem on my LXC serveur with 20Go available and just 1.5Go use for peertube folder in my laptop.

**Comment by rigelk** [comment]
Closing as this seems solved.
